src/any_saves_menu.cpp

```diff
@@ -0,0 +1,488 @@
+#include "libtp_c/include/tp.h"
+#include "libtp_c/include/system.h"
+#include "libtp_c/include/controller.h"
+#include "menu.h"
+#include "controller.h"
+#include "fifo_queue.h"
+#include "utils.h"
+#include "gorge.h"
+#include "rollcheck.h"
+#include "save_injector.h"
+#include "log.h"
+#include "fs.h"
+#define LINES 48
+
+static Cursor cursor = {0,0};
+static CameraMatrix camera = {0, 0, 0, 0, 0, 0, 0, 0};
+static uint16_t angle = 0;
+static Vec3 position = {0, 0, 0};
+bool init_once = false;
+bool any_saves_visible;
+
+Line lines[LINES] = {
+    {"ordon gate clip", ORDON_GATE_CLIP_INDEX, "gate clip outside ordon spring"},
```

Comment:
These lines are using the old descriptions where everything is lowercase. Should copy the values from the latest version.